### PR TITLE
CST-1882: new confirmation screen after adding a post-transitional pa…

### PR DESCRIPTION
**Pull request by ltello**
…rticipant

### Context

- Ticket: [Content updates to cater for post-transitional ECTs](https://dfedigital.atlassian.net/browse/CST-1882)

image

### CODE REVIEW

File: app/views/schools/add_participants/eligibility_confirmation/_post_transitional.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<p class="govuk-body"> We’ll let them know their statutory induction requirement has changed too. </p>
+
+<h2 class="govuk-heading-m"> This person’s statutory induction requirement has changed from 1 year to 2 years </h2>
+<p class="govuk-body"> Our checks show this person began their statutory induction before 1 September 2021. They originally needed to complete a 1-year induction. </p>
+<p class="govuk-body"> ECTs now need to complete a 2-year induction if they started induction before September 2021 and did not complete it by 1 September 2023.</p>
+<p class="govuk-body"> This ECT:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li> does not need to start induction again </li>
+  <li> must complete what remains of their 2-year induction /li>
```
Comment (tonyheadford) [inline app/views/schools/add_participants/eligibility_confirmation/_post_transitional.html.erb:9]: Missing `<`

Comment (ltello) [reply app/views/schools/add_participants/eligibility_confirmation/_post_transitional.html.erb]: Thanks!

**Review by tonyheadford — APPROVED** [review]

File: app/views/schools/add_participants/eligibility_confirmation/_post_transitional.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="govuk-panel govuk-panel--confirmation">
```
Comment (frankieroberto) [inline app/views/schools/add_participants/eligibility_confirmation/_post_transitional.html.erb:1]: This just needs bit more margin beneath it.

```suggestion
<div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-7">
```

**Review by frankieroberto — APPROVED** [review]
One minor tweak from me.

**Pull request merged**